cmd/cerbosctl/enable/enable.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2021-2023 Zenauth Ltd.
```

Comment:
Probably can add this definition to `policy.go` or vice versa. It feels too Java-like to have a single file for a single type 😄